storage/buckets/get_bucket_policy.go

```diff
@@ -14,7 +14,7 @@
 
 package buckets
 
-// [START storage_get_bucket_policy]
+// [START storage_view_bucket_iam_members]
```

Comment:
It looks like the live site still uses the version of this sample from main.go (https://github.com/GoogleCloudPlatform/golang-samples/blob/d9097fd487f91798f4d203a89456adafbfbc890b/storage/buckets/main.go#L167 ) rather than the split one. This one has already been updated to use `Policy3` which is a bit concerning because of inconsistencies. When will this be switched over? @JesseLovelace @frankyn

Reply:
Working on that and will get you a timeline. Thank you for your patience.